### PR TITLE
Minor readme compilation instruction fixes

### PR DESCRIPTION
**Pull request by StanczakDominik**
I sort of did them blind, if you'd like to reword anything, shoot :)

### CODE REVIEW

**Comment by StanczakDominik** [comment]
cc @p2004a :)

File: README.markdown [hunk 1]
```diff
@@ -21,25 +21,38 @@ Detailed instructions for how to compile Recoil can be found [here](https://gith
...
-	git checkout origin/BAR105 -b BAR105
+```bash
+git remote add upstream git@github.com:beyond-all-reason/spring.git
+git fetch --all --tags
+```
+
+Check out the BAR105 branch in this case:
+
+```bash
+git checkout origin/BAR105 -b BAR105
```
Comment (sprunk) [inline README.markdown:47]: You added the remote as "upstream" though?
```suggestion
git checkout upstream/BAR105 -b BAR105
```

Comment (StanczakDominik) [reply README.markdown:47]: Gah, yes, as you can imagine I didn't update this line. Thanks for the catch!

**Review by sprunk — APPROVED** [review]
It's just a git basics tutorial, but it sounds fine to have if it helps somebody

**Comment by StanczakDominik** [comment]
Adding it was explicitly suggested by @p2004a BAR's dev-main discord channel. I do agree it's pretty basic.

**Pull request merged**